moped-editor/src/utils/activityLogFormatters/mopedProjectTypesActivity.js

```diff
@@ -22,7 +22,7 @@ export const formatProjectTypesActivity = (change, projectTypeList) => {
   }
 
   // delete an existing project type
-  if (change.description[0].field === "is_deleted") {
+  if (change.description[0].fields.includes("is_deleted")) {
```

Comment:
This is the deprecated project types that are stored in the `deprecated.moped_proj_types` table. I couldn't find a production project where a project type was removed and will try again in the morning. This should do the trick for any older projects anyways.